### DISCUSSION TITLE
Guest user access

### DISCUSSION THREAD

**Discussion by alasdair-m** [YannickRe/spfx-msgraph-peoplesearch#39] [Q&A] [answered]
Hi, Yannick, I thought I'd put this in discussions as I imagine it's going to be something in my tenant, not a bug. But I'm getting the error "Insufficient privileges to complete the operation" when guest users access the web part. Have you seen this before? And thank you, the people search is excellent. :)

**Comment by YannickRe** [comment, marked as answer]
It does work for Guest Users but only when Guest User permissions are not restricted. It's a setting you can change in https://portal.azure.com => User settings => Manage external collaboration settings
![image](https://user-images.githubusercontent.com/9973962/137799609-419e05dc-b706-4212-8f0f-dbe3e6c6914c.png)

It does not work with any of the other options. The last one will never work, but the second one might be achievable. I have never tried adapting the code to work with that option, and I will probably not attempt in the near future due to lack of time.

Thank you for the nice words, it's always appreciated!

**Reply by alasdair-m** [reply]
Excellent! Thank you!
